test/osb_test/osb_suite_test.go

```diff
@@ -107,7 +108,7 @@ type brokerPlatformCredentials struct {
 
 var _ = BeforeSuite(func() {
 	ctx = common.NewTestContextBuilderWithSecurity().WithEnvPreExtensions(func(set *pflag.FlagSet) {
-		Expect(set.Set("server.request_timeout", timeoutDuration.String())).ToNot(HaveOccurred())
+		// Expect(set.Set("server.request_timeout", timeoutDuration.String())).ToNot(HaveOccurred())
```

Comment:
 should this stay or be removed ?

Reply:
forgot to remove it while testing the tests :D